src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
alive needs to be static ( if the linter drops warning, add eslint-parser in deeps.

Reply:
Discussed it on Q&A. Mentor said don't touch the linter, and my solution is OK.